src/jit/importer.cpp

```diff
@@ -5757,8 +5763,26 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
                                     new (this, GT_CNS_INT) GenTreeIntCon(TYP_INT, pFieldInfo->offset, fs));
             }
             break;
+        }
+        case CORINFO_FIELD_STATIC_READYTORUN_HELPER:
+        {
+            noway_assert(opts.IsReadyToRun());
+            CORINFO_GENERICHANDLE_RESULT embedInfo;
+            info.compCompHnd->embedGenericHandle(pResolvedToken, FALSE, &embedInfo);
+            assert(embedInfo.lookup.lookupKind.needsRuntimeLookup);
 
+            GenTreePtr ctxTree = getRuntimeContextTree(&embedInfo.lookup);
+
+            var_types type = TYP_BYREF;
+            op1            = impReadyToRunHelperToTree(pResolvedToken, pFieldInfo->helper, type, gtNewArgList(ctxTree),
+                                            &embedInfo.lookup.lookupKind);
+            FieldSeqNode* fs = GetFieldSeqStore()->CreateSingleton(pResolvedToken->hField);
```

Comment:
`hField` is the same for both `Foo<String>._s` and `Foo<Object>._s`. Does the FieldSeqNode need to be unique enough to distinguish between these two case?